### ISSUE TITLE
Connecting to kernel stucks

### ISSUE THREAD

**Issue by asandhaa** [spyder-ide/spyder#21219]
### What steps will reproduce the problem?

I use pypsa-eur to run a power system model. This morning I updated the Gurobi version to 10 and then the kernel couldn't connect

What I tried up to now:

- conda update pyzmq
- conda update ipykernel
- spyder --reset

nothing helped

spyder                    5.2.2            py39haa95532_1
spyder-kernels            2.2.1            py39haa95532_0

What can I do?

Anna

## Versions

* Spyder version: 5.2.2 
* Python version: 3.9.12
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0          :  1.4.1 (OK)
chardet >=2.0.0               :  5.1.0 (OK)
cloudpickle >=0.5.0           :  2.2.1 (OK)
cookiecutter >=1.6.0          :  2.3.0 (OK)
diff_match_patch >=20181111   :  20230430 (OK)
intervaltree >=3.0.2          :  3.1.0 (OK)
IPython >=7.6.0;<8.0.0        :  7.31.1 (OK)
jedi >=0.17.2;<0.19.0         :  0.18.2 (OK)
jellyfish >=0.7               :  1.0.0 (OK)
jsonschema >=3.2.0            :  4.18.6 (OK)
keyring >=17.0.0              :  24.2.0 (OK)
nbconvert >=4.0               :  7.7.3 (OK)
numpydoc >=0.6.0              :  1.5.0 (OK)
paramiko >=2.4.0              :  3.3.1 (OK)
parso >=0.7.0;<0.9.0          :  0.8.3 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.9.5 (OK)
pygments >=2.0                :  2.15.1 (OK)
pylint >=2.5.0                :  2.17.5 (OK)
pyls_spyder >=0.4.0           :  0.4.0 (OK)
pylsp >=1.3.2;<1.4.0          :  1.3.3 (OK)
pylsp_black >=1.0.0           :  1.0.0 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.2 (OK)
qtawesome >=1.0.2             :  1.2.3 (OK)
qtconsole >=5.2.1;<5.3.0      :  5.2.2 (OK)
qtpy >=1.5.0                  :  2.3.1 (OK)
rtree >=0.9.7                 :  1.0.1 (OK)
setuptools >=49.6.0           :  59.8.0 (OK)
sphinx >=0.6.6                :  7.1.2 (OK)
spyder_kernels >=2.2.1;<2.3.0 :  2.2.1 (OK)
textdistance >=4.2.0          :  4.5.0 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3             :  3.0.0 (OK)
zmq >=17                      :  25.1.0 (OK)

# Optional:
cython >=0.21                 :  None (NOK)
matplotlib >=2.0.0            :  None (NOK)
numpy >=1.7                   :  1.21.5 (OK)
pandas >=1.1.1                :  1.3.5 (OK)
scipy >=0.17.0                :  1.7.3 (OK)
sympy >=0.7.3                 :  None (NOK)
```

**Comment by ccordoba12** [comment]
Hey @asandhaa, thanks for reporting. You said:

> What I tried up to now:
>
>    conda update pyzmq
>    conda update ipykernel

I'm sorry to say but that was not helpful because the latest versions of PyZMQ and IPykernel and probably not compatible with the version of Spyder-kernels you're using.

So, I recommend you to update to update our latest version available in Anaconda (5.4.3). To do that, please close Spyder, open the Anaconda Prompt and run there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).

Let us know if that works for you.

**Comment by ccordoba12** [comment]
Closing due to lack of response.
